Week-3/3.3.js

```diff
@@ -0,0 +1,22 @@
+function createIncrement() 
+{
+    let count=0; 
+    function increment() { 
+        count++; 
+    }
+    let message = `Count is ${count}`; 
+    function log() { 
+        console.log(message);
+    }
+    return [increment,log];
+}
+const [increment,log] = createIncrement();
+increment();
+increment();
+increment();
+log();// What is logged?
+
+// Count is 0 because when the createIncrement function is called, the message variable is 
+// already declared and set to string "Count is 0" since during that time count was 0, but 
```

Comment:
Correct.

What if we use var or const instead of let?Or can you elaborate why it is already declared